test/functions/diff.js

```diff
@@ -19,10 +19,19 @@ test('diff versions test', (t) => {
     ['1.1.0', '1.1.0-pre', 'minor'],
     ['1.1.0-pre-1', '1.1.0-pre-2', 'prerelease'],
     ['1.0.0', '1.0.0', null],
+    ['1.0.0-1', '1.0.0-1', null],
     ['0.0.2-1', '0.0.2', 'patch'],
+    ['0.0.2-1', '0.0.3', 'patch'],
+    ['0.0.2-1', '0.1.0', 'minor'],
+    ['0.0.2-1', '1.0.0', 'major'],
     ['0.1.0-1', '0.1.0', 'minor'],
     ['1.0.0-1', '1.0.0', 'major'],
-    ['0.0.0-1', '0.0.0', 'prerelease'],
+    ['1.0.1-1', '1.0.1', 'patch'],
+    ['0.0.0-1', '0.0.0', 'major'],
+    ['1.0.0-1', '2.0.0', 'major'],
```

Comment:
Again this is two major `.inc` operations but that's still a major diff

```
> semver.inc('1.0.0-1', 'major')
'1.0.0'
> semver.inc('1.0.0', 'major')
'2.0.0'
```

Reply:
these are `pre*` now